torch/_dynamo/codegen.py

```diff
@@ -157,7 +157,7 @@ def __call__(self, value, allow_cache=True):
         self.top_of_stack = value
 
     def add_graph_output(self, value):
-        graph_outputs_key = id(value.proxy)
+        graph_outputs_key = id(value.as_proxy())
```

Comment:
does this... do anything lol

Reply:
Its the better convention, but no, should be same.